libs/hdf-converters/src/xccdf-results-mapper.ts

```diff
@@ -278,142 +299,149 @@ export class XCCDFResultsMapper extends BaseConverter {
               },
               nist: {
-                path: 'Rule.ident',
+                path: ['ident', 'reference'],
                 transformer: nistTag
               },
-              severity: {path: 'Rule.severity'},
+              severity: {path: 'severity'},
               description: {
-                path: ['Rule.description.text', 'Rule.description'],
-                transformer: (description: string) =>
-                  JSON.stringify(
-                    _.pickBy(convertEncodedXmlIntoJson(description), _.identity)
-                  )
+                path: ['description.text', 'description'],
+                transformer: (description: string): string =>
+                  _.get(
+                    parseXml(description),
+                    'VulnDiscussion',
+                    description
+                  ) as string
               },
-              group_id: {path: 'id'},
-              group_title: {path: ['title.text', 'title']},
+              group_id: {path: 'group.id'},
+              group_title: {path: ['group.title.text', 'group.title']},
               group_description: {
-                path: ['description.text', 'description'],
-                transformer: (description: string) =>
-                  JSON.stringify(
-                    _.pickBy(convertEncodedXmlIntoJson(description), _.identity)
-                  )
+                path: ['group.description.text', 'group.description'],
+                transformer: (description: string): string =>
+                  _.get(
+                    parseXml(description),
+                    'GroupDescription',
+                    description
+                  ) as string
               },
-              rule_id: {path: 'Rule.id'},
-              check: {path: 'Rule.check'},
-              fix_id: {path: 'Rule.fix.id'},
+              rule_id: {path: 'id'},
+              check: {path: 'check'},
+              fix_id: {path: 'fix.id'},
               fixtext_fixref: {
-                path: 'Rule.fixtext.fixref'
+                path: ['fixtext.fixref.text', 'fixtext.fixref']
               },
-              ident: {path: 'Rule.ident'},
-              reference: {path: 'Rule.reference'},
-              selected: {path: 'Rule.selected'},
-              version: {path: 'Rule.version.text'},
-              weight: {path: 'Rule.weight'},
-              profiles: {
-                path: '$.Benchmark.Profile',
-                transformer: (
-                  profiles: Record<string, unknown>[]
-                ): Record<ProfileKey, unknown>[] => {
-                  const pathsSelect = ['select'];
-                  const paths = {
-                    id: ['id'],
-                    description: ['description.text', 'description'],
-                    title: ['title.text', 'title']
-                  };
-                  return getProfiles(profiles, pathsSelect, paths);
+              ident: {path: 'ident'},
```

Comment:
![image](https://github.com/mitre/heimdall2/assets/2015821/05088c26-64a7-41fb-a048-ed2ebbc621f0)

seems like there are more attributes that don't exist in the output coming from complianceascode.  we should probably toss those in the transformer as a conditionally provided attribute (see the version example).

Reply:
I believe I have now address this point.